book/website/collaboration/inclusive-events.md

```diff
@@ -31,9 +31,12 @@ It covers:
 (cl-inclusive-events-background)=
 ## Motivation and Background
 
-Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background. 
-This chapter is meant to serve as a resource to support and guide our community members with the tools to identify the opportunities and challenges in upholding this, and to approach event planning and design with accessibility and inclusion in mind.
+Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background and there has been an increased effort in many disciplines to involve members of the public and people with lived experience in research projects. 
```

Comment:
It might be a personal preference, but I think this opening is very broad, going from "un declaration of human rights"-level of ambition straight into PPIE efforts. I think it would be good to either have a bit more connecting bits between those two, or alternatively replace the first half with a bit more PPIE centric background on why PPIE efforts are now a thing & what that means for events.

Reply:
I've reworked the introduction in the latest commit. I was struggling to find a way to introduce the topic in the landing page without having to write a whole separate section explaining PPIE's purpose and methods in general.

If you think this is a better way to introduce the topic then happy to keep the change - it may be something that can be revised by a collaborator as I've been really stuck on the landing page's introduction.

Reply:
I think the updated version feels very good to me!